userdocs/profiles.dev/docs/installer-docs/generate-local-manifests.md

```diff
@@ -30,3 +30,52 @@ pctl install \
   --out relative-path \
   <catalog name>/<profile>
 ```
+
+Consider the following installation:
+
+```
+pctl install --git-repository flux-system/flux-system nginx-catalog/nginx/v2.0.1
+generating profile installation from source: catalog entry nginx-catalog/nginx/v2.0.1
+installation completed successfully
+```
+
+Let's take a look inside:
+
+```
+tree nginx
+nginx
+├── artifacts
+│   └── bitnami-nginx
+│       ├── helm-chart
+│       │   ├── ConfigMap.yaml
+│       │   ├── HelmRelease.yaml
+│       │   └── HelmRepository.yaml
+│       ├── kustomization.yaml
+│       └── kustomize-flux.yaml
+└── profile-installation.yaml
+```
+
+This profile installs an `nginx` using a remote helm chart. The folders contain the following files in order:
+
+* bitnami-nginx - the name of the artifact
+* helm-chart - contains the resources which install the actual nginx
+* ConfigMap.yaml - contains any settings which the artifact had
+* HelmRelease.yaml - contains the HelmRelease object which installs nginx
+* HelmRepository.yaml - contains the definition of the helm chart repository where the chart is located at
+* kustomization.yaml - this is a file which tells flux what to install -- more on this later
```

Comment:
"more on this later" -> where is this more? is it the bit at the bottom? maybe "see below" is easier?

Reply:
oh yes. I wanted to fix this reference. cheers.